WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -119,25 +119,29 @@ def update_all
           RegistrationsMailer.notify_registrant_of_accepted_registration(registration).deliver_now
         end
       end
-      flash[:success] = "#{"Registration".pluralize(registrations.length)} accepted! Email #{"notification".pluralize(registrations.length)} sent."
+      flash.now[:success] = "#{"Registration".pluralize(registrations.length)} accepted! Email #{"notification".pluralize(registrations.length)} sent."
```

Comment:
This particular form that @jonatanklosko changed required javascript to work anyways. See <https://staging.worldcubeassociation.org/competitions/IranOpen2016/edit/registrations>. You have to check some registrations before the option to accept/reject their registrations even shows up.